datasets/README.md

```diff
@@ -16,6 +16,35 @@ This directory contains small public datasets in `mtx` and `csv` format used by
 
 **netscience** : The graph netscience contains a coauthorship network of scientists working on network theory and experiment, as compiled by M. Newman in May 2006.
 
+
+
+## Tests coverage 

```

Comment:
In this PR, I updated  `datasets/README.md` to reference and describe the new datasets. But after discussing with Rick, I instead modified the existing datasets (karate, dolphins) since they are much smaller, therefore easier to test. I will be uploading the new datasets in this PR